src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -127,7 +142,7 @@ const PathwaysListElement: FC<PathwaysListElementProps> = ({
                   <th>patient elements</th>
                 </tr>
                 {criteria &&
-                  criteria.map(c => (
+                  criteria.criteriaResultItems.map(c => (
```

Comment:
for some reason I can't get multiline suggestion to work, but:

`criteria?.criteriaResultItems?.map`

Reply:
Updated here and line 61/62